### DISCUSSION TITLE
functions recently fail due to "esbuild-linux-64" package not being found

### DISCUSSION THREAD

**Discussion by hassanzadeh** [vercel/vercel#6865] [Help]
Hey guys,
Starting form about a week ago, all my deployments fail at Run time with this error (copied form functions log):
"2021-10-19T17:58:53.710Z	f7284007-d328-48a7-b4e3-ecf669500227	ERROR	Error: The package "esbuild-linux-64" could not be found, and is needed by esbuild.
If you are installing esbuild with npm, make sure that you don't specify the
"--no-optional" flag. The "optionalDependencies" package.json feature is used
by esbuild to install the correct binary executable for your current platform.
    at generateBinPath (/var/task/node_modules/esbuild/lib/main.js:1643:15)"

I'm not sure what has changed under the hood that is causing everything to break. I also noticed that I'm getting these warnings during the build time;
Info esbuild-darwin-arm64@0.13.8: The platform "linux" is incompatible with this module.

Any thoughts?
thanks

**Comment by dominik-sfl** [comment]
Same problem. For me, everything works fine on my local machine (both dev & build), but once I deploy to Vercel I get the same error, but only server side (calling esbuild/mdxbuild in `getStaticProps` works fine for me).  I ran into this problem using `mdx-bundler` which requires `esbuild` to be installed.

I ran into the issue with both `npm` & `yarn`, and created reproduction repos here:

reproduction repo using yarn: https://github.com/dominik-sfl/next-mdx-bundler
reproduction repo  using npm: https://github.com/dominik-sfl/next-mdx-bundler-npm

live demo (yarn) : https://next-mdx-bundler.vercel.app/
live demo (npm) : https://next-mdx-bundler-npm.vercel.app/

Downgrading `esbuild` to `0.12.09` fixed things for me for now, but not sure what could be done to get it working with the latest version?
